index.bs

```diff
@@ -439,6 +441,8 @@ When this method is invoked, the user agent MUST execute the following algorithm
     - If <a>allowList</a> is undefined or empty, let |credentialList| be an empty list. Otherwise, execute a platform-specific
         procedure to determine which, if any, credentials listed in <a>allowList</a> might be present on this authenticator, and
         set |credentialList| to this filtered list. If no such filtering is possible, set |credentialList| to an empty list.
+    - For each credential C within the |credentialList| that has a non-empty |transports| list, use only the specified transports
+        to get assertions using credential C.
```

Comment:
Also needs more indent (see Travis-CI error).

Also might want to soften this so the hint is not mandatory for the client.